frontend/src/metabase/lib/icon.ts

```diff
@@ -57,4 +59,9 @@ export const getIconBase = (
   return { name: modelIconMap?.[item.model] ?? "unknown" };
 };
 
-export const getIcon = PLUGIN_COLLECTIONS.getIcon ?? getIconBase;
+export const getIcon = (item: ObjectWithModel, options: IconOptions = {}) => {
+  if (PLUGIN_COLLECTIONS) {
```

Comment:
Should we also take [`PLUGIN_MODERATION` into account](https://github.com/metabase/metabase/blob/0e70276/frontend/src/metabase/entities/questions.js#L183-L191)?

It also seems that [`getCollectionIcon` has not been fully ported](https://github.com/metabase/metabase/blob/e57313a1f41d1a906669ea18860015835d9d5739/frontend/src/metabase/entities/collections/utils.ts#L18-L31).